### ISSUE TITLE
Rule triggered on variable declarations in Less

### ISSUE THREAD

**Issue by bdukes** [sh-waqar/stylelint-declaration-use-variable#7]
I have an <a href="http://www.sscce.org/"><abbr title="Short, Self Contained, Correct (Compilable), Example">SSCCE</abbr></a> at https://github.com/bdukes/stylelint-variable-less-sscce.

When using a rule definition like `"declaration-use-variable": [ "/color/" ]`, this rule is triggered by assigning values to variables which have the name `color` in them.  Obviously we don't want the assignment to a variable to be marked as not using a variable.

Hope it helps!

**Comment by msaforrian** [comment]
I was having the same problem. Here's a fancy regex that excludes Sass/Less variables

```regex
(^[^@$].*(color).*|^color)
```

Here's a [regex tester demonstrating](http://www.regexpal.com/?fam=95206) it.
